cmd/main.go

```diff
@@ -170,6 +171,13 @@ func main() {
 		ManagedDomainsAPI:   domainHandler,
 		InnerMiddleware:     metrics.WithMatchedRoute(log.WithField("pkg", "matched-h"), prometheusRegistry),
 	})
+	if Options.Auth.AllowedDomains != "" {
+		allowedDomains := strings.Split(strings.ReplaceAll(Options.Auth.AllowedDomains, " ", ""), ",")
```

Comment:
why do we need remove spaces? :)

Reply:
cause u can set "openshift.com, *"
and "*" will not work as u expect, u need to trim spaces.
Already felt on this